### PR TITLE
Nodes for Path of Travel element

### PR DESCRIPTION
**Pull request by jrichter2019**
### Purpose

Adding nodes for new 2020 functionality - Path of Travel

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@QilongTang 

@AndyDu1985 @alfarok 

### FYIs

@awbushnell

### CODE REVIEW

File: src/Libraries/RevitNodes/Elements/PathOfTravel.cs [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+      /// <param name="startPtList">List of start points</param>
+      /// <param name="endPtList">List of end points</param>
+      /// <param name="manyToMany">If true, paths are created from every point in the start point list to all points in the end point list. If false, a path is created from every point in the start point list to a corresponding point in the end point list with the same index. The two lists must have the same size when not creating many-to-many paths.</param>
+      /// <returns>List of Path of Travel elements; can contain null elements if there is no path between some points.</returns>
+      [NodeCategory("Create")]
+      [AllowRankReduction]
+      public static PathOfTravel[] ByFloorPlanPoints(Revit.Elements.Views.FloorPlanView floorPlan, Autodesk.DesignScript.Geometry.Point[] startPtList, Autodesk.DesignScript.Geometry.Point[] endPtList, bool manyToMany)
+      {
+         if (floorPlan == null)
+            throw new ArgumentNullException("Invalid view");
```
Comment (mjkkirschner) [inline src/Libraries/RevitNodes/Elements/PathOfTravel.cs]: these strings should get added to the .resx files in this same project so they can be localized by the localization team :)

Comment (jrichter2019) [reply src/Libraries/RevitNodes/Elements/PathOfTravel.cs]: 🤦‍♂

**Comment by jrichter2019** [comment]
@mjkkirschner, would you take another look at this? Does it look good enough to be merged?

**Comment by mjkkirschner** [comment]
Hi @jrichter2019 it looks good to me - but I think you'll need sign off from @AndyDu1985 or @ZiyunShang to merge.

But -  I do see there are no integration tests - which open revit and a .dyn file together.
here is an example for directShape:
https://github.com/DynamoDS/DynamoRevit/blob/master/test/Libraries/RevitIntegrationTests/DirectShapeTests.cs

**Comment by AndyDu1985** [comment]
@jrichter2019 will add integration tests in another PR.

**Comment by QilongTang** [comment]
@jrichter2019 @AndyDu1985 @mjkkirschner  Also seems missing node icon unless I missed something

**Comment by jrichter2019** [comment]
@QilongTang - yes, we are fine with the default icon if it's ok with you.

**Comment by QilongTang** [comment]
@jrichter2019 Sure, just confirming this is by intention.

**Pull request merged**